terraform/eks/variables.tf

```diff
@@ -81,6 +81,21 @@ variable "configuration_source" {
   }
 }
 
+variable "is_inject_auto_instrumentation" {
+  type    = bool
+  default = false
+}
+
+variable "java_auto_instrumentation_repository" {
+  type    = string
+  default = "public.ecr.aws/aws-observability/adot-autoinstrumentation-java"
+}
+
+# latest tag is not available
+variable "java_auto_instrumentation_tag" {
```

Comment:
where are you setting the value for this variable?

Reply:
@rapphil as per [this](https://github.com/aws-observability/aws-otel-test-framework/pull/1283#discussion_r1242461105) comment from Bryan, I have removed the default value of this variable. So when running this testcase manually the terraform prompts to provide this variable value. But we are passing the value of this variable from [here](https://github.com/aws-observability/aws-otel-java-instrumentation/blob/4c42d3a4dbe418b01d2e1c384bea0f0c4a93367a/.github/workflows/main-build.yml#L255) in mainbuild workflow in java-instrumentation repo.